content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,256 @@
+   starts. As a result, they manually enter a greater demand in the |MPS|, so they can properly
+   replenish the product in time for the increase in customer demand.
+
+.. important::
+   It is essential to remember that the |MPS| is a **MANUAL** tool. Adding a product to the |MPS|
+   does not cause it to be manufactured or purchased automatically. The |MPS| simply suggests the
+   amount of the product that should be replenished, but requires user input to create the |MOs| or
+   purchase orders |POs| that are used to replenish it.
+
+   For this reason, it is recommended that the |MPS| **NOT** be used alongside reordering rules for
+   the same product. Because reordering rules are an automated workflow, they conflict with the
+   manual replenishment method of |MPS|. Using both in unison can lead to inaccurate forecasts and
+   the creation of unnecessary replenishment orders.
+
+Enable and configure MPS
+========================
+
+To use the |MPS| feature, navigate to :menuselection:`Manufacturing app --> Configuration -->
+Settings`, and tick the :guilabel:`Master Production Schedule` checkbox under the
+:guilabel:`Planning` heading. Finally, click :guilabel:`Save`.
+
+After enabling the :guilabel:`Master Production Schedule` feature, two new fields appear under it on
+the :guilabel:`Settings` page: :guilabel:`Time Range` and :guilabel:`Number of Columns`.
+
+The :guilabel:`Time Range` field is used to select the period of time over which planning takes
+place, and offers three options: :guilabel:`Monthly`, :guilabel:`Weekly`, and :guilabel:`Daily`. For
+example, if :guilabel:`Monthly` is selected, the |MPS| plans the production requirements of products
+and components on a monthly basis.
+
+The :guilabel:`Number of Columns` field is used to specify the quantity of the selected
+:guilabel:`Time Range` units shown on the |MPS| page. For example, if the :guilabel:`Time Range`
+field is set to :guilabel:`Monthly`, and `12` is entered in the :guilabel:`Number of Columns` field,
+the |MPS| shows one column for the next 12 months, starting with the current month.
+
+If the values of the :guilabel:`Time Range` or :guilabel:`Number of Columns` fields are altered,
+click :guilabel:`Save` again to save the changes.
+
+.. image:: use_mps/mps-setting.png
+   :align: center
+   :alt: The MPS setting in the Manufacturing app's settings.
+
+MPS UI
+======
 
-Go to the :menuselection:`Manufacturing app --> Configuration --> Settings` and
-activate the Master Production Schedule feature before hitting save.
+To open the |MPS|, navigate to :menuselection:`Manufacturing app --> Planning --> Master Production
+Schedule`. The |MPS| view appears as follows:
+
+.. image:: use_mps/mps.png
+   :align: center
+   :alt: The Master Production Schedule in the Manufacturing app.
+
+The grey column on the left side of the screen shows a section for every product added to the |MPS|.
```

Comment:
While what you have is accurate, I have another perspective to share (:

Instead of just referring to it as a grey column, consider describing it within the context of the entire MPS. For example:

The master production schedule is composed of rows, each containing a product with breakdowns of demand, suggested replenishment, and resulting forecasted stock. The products in the grey column depend on the filters. The columns of the MPS depend on the time range and number of column configurations in the :ref:`settings <line 63>`